### PR TITLE
Final FI of dev/eng before RI to master

### PR DESCRIPTION
**Pull request by joperezr**
This is to prevent any conflicts of the RI

cc: @weshaggard @ericstj @mellinoe @chcosta @karajas

### CODE REVIEW

File: src/System.Diagnostics.Debug/src/Configurations.props [hunk 1]
```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Unix;
       netstandard-Windows_NT;
+      netcoreapp1.2corert-Unix;
+      netcoreapp1.2corert-Windows_NT;
```
Comment (chcosta) [inline src/System.Diagnostics.Debug/src/Configurations.props]: I don't see anything OS specific w.r.t. netcoreapp1.2corert in the csproj file.

Comment (joperezr) [inline src/System.Diagnostics.Debug/src/Configurations.props]: There is a condition for GenFacadesArgs in the .csproj that got added.
```xml
<GenFacadesArgs Condition="'$(TargetGroup)' == 'netcoreapp1.2corert'">$(GenFacadesArgs) -ignoreMissingTypes</GenFacadesArgs>
```

Comment (chcosta) [inline src/System.Diagnostics.Debug/src/Configurations.props]: Yes, but nothing OS specific.  ie, the buildconfigurations entry could just be `netcoreapp1.2corert`, unless there is something intrinsic to GenFacadesArgs which requires an OS be targeted.  From a cursory glance, I don't see anything.

Comment (joperezr) [inline src/System.Diagnostics.Debug/src/Configurations.props]: That I agree, but this is how it was added in master so I'm just replicating it. I will have to do a pass on this anyway on master, so for now I'll check in this as is, and go and fix all of the netcoreapp1.2corert configurations as part of the UWP work.

**Pull request merged**